controller/api/models.py

```diff
@@ -179,6 +179,28 @@ def __str__(self):
     def url(self):
         return self.id + '.' + settings.DEIS_DOMAIN
 
+    def _get_job_id(self, container_type):
+        app = self.id
+        release = self.release_set.latest()
+        version = "v{}".format(release.version)
+        job_id = "{app}_{version}.{container_type}".format(**locals())
+        return job_id
+
+    def _get_command(self, container_type):
```

Comment:
this code is identical to the Container class' implementation. If we're already knowledgeable about the container's process type, why aren't we calling the container's function instead?